cmd/server-mux.go

```diff
@@ -457,6 +460,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 			// Execute registered handlers, update currentReqs to keep
 			// tracks of current requests currently processed by the server
 			atomic.AddInt32(&m.currentReqs, 1)
+			lmsg := fmt.Sprintf("serving for %s", r.URL.String())
+			log.logger.Info(lmsg)
```

Comment:
Thanks harshavardhana, that looks much better than what I have and I will try to deploy it internally next cycle.

Sorry balamurugana, I am not quite sure how to read what you are saying. I understand opposing interim solutions, but we have product to ship.